templates/default/rspec_env.erb

```diff
@@ -1,6 +1,6 @@
 # Copy this to .env and update values
 WEB_HOST=localhost                                        # Web host local or remote
-WEB_PORT=8181                                             # Web port local or remote
+WEB_PORT=<%= @hw_port %>                                  # Web port local or remote
```

Comment:
Does the load balancer port work on the single VM (ubuntu) tests?

Reply:
Never mind I see it changes it to the https port in case of single elastic setup.